extensions/wikipedia/package.json

```diff
@@ -40,9 +41,44 @@
     "Web",
     "News"
   ],
+  "preferences": [
+    {
+      "name": "zhVariant",
+      "title": "Chinese Language Variants",
+      "description": "Select the default language variant for the Chinese Wikipedia",
+      "required": false,
+      "type": "dropdown",
+      "data": [
+        {
+          "title": "臺灣正體",
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.

Reply:
Hi, @pernielsentikaer, I think the intention of this preference is not to localize the extension, but just to support the original behavior of Wikipedia.

<img width="1258" alt="image" src="https://user-images.githubusercontent.com/38185674/216983165-3ce1cb2c-3c49-49cb-a545-b9d8cc4f4faa.png">

As you can see, there is a dropdown at the top of the Chinese Wikipedia to support language variants. The Wikipedia extension has a dropdown to support searching in different language already, but I can't think of any easy way to add language variants support inside that dropdown.

And I think just like the [**translate** extension](https://github.com/raycast/extensions/blob/main/extensions/google-translate/package.json#L34), although Raycast doesn't support localization, there can be preferences allowing the user to select their language to use this extension.

Or do you have any other suggestions to support the language variants feature of the Wikipedia extension?

Reply:
You can have the preferences just fine, but you need their `title`'s to be in US English 😊